client/src/components/ChatPreview.tsx

```diff
@@ -9,7 +9,9 @@ export interface IChatPreviewProps {
 export default class ChatPreview extends React.Component<IChatPreviewProps> {
   public render() {
     return (
-      <div>
+      <div className='chat_preview'>
+        <div className='user_pic'>{this.props.user.avatarUrl}</div>
```

Comment:
Use this as an actual url either in `<img/>` or in `background-image`

Reply:
done